common/tools/dev-tool/src/commands/run/build-package.ts

```diff
@@ -13,16 +14,22 @@ export const commandInfo = makeCommandInfo("build-package", "build a package for
 const TSHY_BIN_PATH = path.resolve(__dirname, "..", "..", "..", "node_modules", ".bin", "tshy");
 
 export default leafCommand(commandInfo, async () => {
-  log.info(`Building package with tshy from ${TSHY_BIN_PATH}`);
-
-  await concurrently(
-    [
-      {
-        command: TSHY_BIN_PATH,
-      },
-    ],
-    { raw: true },
-  ).result;
+  const commandPath = isWindows() ? `${TSHY_BIN_PATH}.CMD` : TSHY_BIN_PATH;
```

Comment:
I kinda just use concurrently to handle the windows / other OS's extension 🙈 

that's probably why I have a call to concurrently with just one command.